data/net/minecraft/world/item/ItemStack.mapping

```diff
@@ -74,13 +74,13 @@ CLASS net/minecraft/world/item/ItemStack
 		ARG 1 blockRegistry
 		ARG 2 block
 	METHOD hasCustomHoverName ()Z
-		COMMENT Returns true if the itemstack has a display name
+		COMMENT Returns {@code true} if the itemstack has a display name
 	METHOD hasTag ()Z
-		COMMENT Returns true if the ItemStack has an NBTTagCompound. Currently used to store enchantments.
+		COMMENT Returns {@code true} if the ItemStack has an NBTTagCompound. Currently used to store enchantments.
 	METHOD hideTooltipPart (Lnet/minecraft/world/item/ItemStack$TooltipPart;)V
 		ARG 1 part
 	METHOD hurt (ILnet/minecraft/util/RandomSource;Lnet/minecraft/server/level/ServerPlayer;)Z
-		COMMENT Attempts to damage the ItemStack with par1 amount of damage, If the ItemStack has the Unbreaking enchantment there is a chance for each point of damage to be negated. Returns true if it takes more damage than getMaxDamage(). Returns false otherwise or if the ItemStack can't be damaged or if all points of damage are negated.
+		COMMENT Attempts to damage the ItemStack with par1 amount of damage, If the ItemStack has the Unbreaking enchantment there is a chance for each point of damage to be negated. Returns {@code true} if it takes more damage than getMaxDamage(). Returns false otherwise or if the ItemStack can't be damaged or if all points of damage are negated.
```

Comment:
```suggestion
		COMMENT Attempts to damage the {@code ItemStack} with {@code amount} damage, If the {@code ItemStack} has the Unbreaking enchantment there is a chance for each point of damage to be negated. Returns {@code true} if it takes more damage than {@code getMaxDamage()}. Returns {@code false} otherwise or if the ItemStack can't be damaged or if all points of damage are negated.
```